deploy/bin/update-worker-types.js

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env node
+
+const bluebird = require('bluebird');
+const workerTypes = require('../etc/worker-types.json');
+const amis = require('../../docker-worker-amis.json');
+const jsonfile = bluebird.Promise.promisifyAll(require('jsonfile'));
+const utils = require('./worker_type_utils');
+
+/*
+ * Write worker-types configuration to a json file
+ */
+async function backupWorkerTypes(client) {
+  console.log('Creating worker-types backup file.');
+  const names = await client.listWorkerTypes();
+  const workerTypes = names.map(name => {
```

Comment:
to avoid confusion, could this be named differently than the JSON file you imported?

Reply:
I renamed the imported json file.